source/index.rst

```diff
@@ -136,23 +134,20 @@ application code, and the serialization support is generated at runtime.
 `eProsima Discovery Server`_
 ----------------------------------------
 
-**eProsima Discovery Server** is an application that simplifies the use and testing of the new `Fast RTPS`_
+*eProsima Discovery Server* is an application that simplifies the use and testing of the new `eProsima Fast DDS`_
 discovery capabilities.
-Now Fast RTPS allows a centralized, non-standard, client-server discovery mechanism that:
+*eProsima Fast DDS* allows a centralized, non-standard, client-server discovery mechanism that:
 
-- extends Fast RTPS use to non-unicast network transports like TCP.
+- extends *eProsima Fast DDS* use to non-unicast network transports like TCP, and
 - speeds up the discovery-stage in a large number of participants scenarios.
```

Comment:
This list is not indented

Reply:
It looks like this because that's where the image ends